### ISSUE TITLE
set-output is deprecated for removal 

### ISSUE THREAD

**Issue by ben-manes** [sibiraj-s/action-archiver#2]
Please update your usage or this action won’t work anymore soon.

> Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

To be safe and not set expectations, I decided to replace my usage with shell. This worked for my need, in case it helps someone else if you don't get to this.

```yaml
# original
with:
  path: '**/{jacoco,results}/**/*.{exec,xml}'
  output: ${{ env.ARTIFACT_NAME }}.tar.gz
  format: tar
  gzip: true

# replacement
run: >
  find . -path */jacoco/*.exec -o -path */results/*.xml
  | tar czf ${{ env.ARTIFACT_NAME }}.tar.gz --files-from -
```

**Comment by sibiraj-sr** [comment]
Should be fixed in v1.1.2. Thanks @ben-manes

**Comment by sibiraj-s** [comment]
Should be fixed in v1.1.2. Thanks @ben-manes
